src/patterns.js

```diff
@@ -99,6 +99,30 @@ module.exports = [
     replacement: '$1var monthNamesShort = <siteinfo-custom-shortMonthNames>;\n$1$2monthNamesShort$3',
     summary: 'wgMonthNamesShort is deprecated ([[phab:T219340|T219340]])'
   },
+  {
+    // wgEnableAPI and wgEnableWriteAPI - part 1 - mw.config.get
+    regex: /([^!]|^)(?:window\.)?mw\.config\.get\(\s*['"]wgEnable(?:Write)?API['"]\s*\)/,
```

Comment:
I've changed this to `(?:wgEnableAPI|wgEnableWriteAPI)` so that it is easier to read for me.